docs/configuration/config-file-reference.md

```diff
@@ -589,6 +589,13 @@ query_scheduler:
   # CLI flag: -query-scheduler.max-outstanding-requests-per-tenant
   [max_outstanding_requests_per_tenant: <int> | default = 100]
 
+  # If a querier disconnects without sending notification about graceful
+  # shutdown, the query-scheduler will keep the querier in the tenant's shard
+  # until the forget timeout expires. This feature is useful to reduce the blast
+  # radius when shuffle-sharding is enabled.
+  # CLI flag: -query-scheduler.querier-forget-timeout
+  [querier_forget_timeout: <duration> | default = 0s]
```

Comment:
Do you think querier_forget_delay or querier_forget_wait would be a more straightforward name for this querier_forget_timeout seems to imply it's trying to do something until it gives up, as opposed to adding a delay to something it could do immediately.

Reply:
Sure. I've renamed it to `querier_forget_delay`.